RFCs/0002-add-pdk-config.md

```diff
@@ -205,10 +206,23 @@ Implement a new pdk subcommand, `pdk config` which will offer the following acti
 
 ### Proposed Configuration Keys
 
+#### System Config
+
+The values for these config keys will be persisted in `/opt/puppetlabs/pdk/config` (`%ProgramData%\PuppetLabs\PDK` on Windows).
```

Comment:
Are these files going to be writable by unprivileged users or are we planning on erroring out if a user tries `pdk set config system.module_defaults.template.url something`?

Reply:
My preference would be to throw an "Access Denied" error.   Hopefully with some nice text

> Are these files going to be writable by unprivileged users

That would be a user decision/policy. Ideally, I wouldn't think so. But the PDK should be able to modify those files if needed.

Reply:
i.e.

```
> pdk set config system.module_defaults.template.url https://somewhere/something
PDK (ERROR) Access Denied. Blah blah

> sudo pdk set config system.module_defaults.template.url https://somewhere/something
{
  system.module_defaults.template.url=https://somewhere/something
}
```

Reply:
sounds good :+1: